website/views/shared/profiles/achievements.jade

```diff
@@ -52,35 +52,35 @@ div(ng-if='::profile.backer.tier')
   small=env.t('kickstartText')
   hr
 
-div(ng-if='profile.achievements.streak || user._id == profile._id')
+div(ng-if='::profile.achievements.streak || user._id == profile._id')
   .achievement.achievement-thermometer(ng-if='::profile.achievements.streak')
   div(ng-class='::{muted: !profile.achievements.streak}')
-    h5(ng-show='profile.achievements.streak > 1 || !profile.achievements.streak')
+    h5(ng-if='::profile.achievements.streak > 1 || ::!profile.achievements.streak')
```

Comment:
Not sure it's needed here twice, just at the beginning of the `ng-if`, like this 
```
ng-if='::profile.achievements.streak > 1 || !profile.achievements.streak'
```

I'll double check and get back to you.